src/pybind/mgr/dashboard/frontend/src/app/shared/api/cephfs-snapshot-schedule.service.ts

```diff
@@ -19,14 +19,38 @@ export class CephfsSnapshotScheduleService {
     return this.http.post(`${this.baseURL}/snapshot/schedule`, data, { observe: 'response' });
   }
 
-  update(data: Record<string, any>): Observable<any> {
+  update({fs, path, ...rest}: Record<string, any>): Observable<any> {
```

Comment:
what is `rest` refers to here?

Reply:
I am destructuring the arg object to grab the properties I need and the rest (which is only the remaining properties) I pass in the body of the http request.

More info below:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/rest_parameters
https://dev.to/coderjay06/how-to-destructure-nested-javascript-objects-1dcl